velox/expression/fuzzer/SparkExpressionFuzzerTest.cpp

```diff
@@ -54,7 +54,8 @@ int main(int argc, char** argv) {
       "chr",
       "replace",
       "might_contain",
-      "unix_timestamp"};
+      "unix_timestamp",
+      "raise_error"};
```

Comment:
This change could be removed if we are throwing user error because user error is caught in the fuzzer test.